### ISSUE TITLE
Patch existing APIs to grant access

### ISSUE THREAD

**Issue by tcheronneau** [Axway-API-Management-Plus/apim-cli#153]
**User story**
I want to authorize a new organization to use an existing API without redeploying API. 

**Additional context**
In the UI you can grant access to selected organization without redeploying or unpublish the API. 
It could be really useful to be able to do that within this cli.

**Comment by cwiechmann** [comment]
Would you expect that command as part of the organization or api command section?

Here an example how it would look like for organizations:
```
Available commands:
apim org delete          Delete selected organizatio(s) from the API-Manager
apim org get             Get Organizations from API-Manager in different formats
apim org import          Import organizatio(s) into the API-Manager
apim org grant-access    Give selected organizations access to one or more APIs
```

And for APIs:
```
Available commands:
apim api import          Import APIs into the API-Manager
apim api delete          Delete the selected APIs from the API-Manager
apim api publish         Publish the selected APIs
apim api change          Changes the selected APIs according to given parameters
apim api get             Get APIs from API-Manager in different formats
apim api upgrade-access  Upgrades access for one or more APIs based on a given 'old/reference' API.
apim api approve         Approves selected APIs that are in pending state
apim api unpublish       Unpublish the selected APIs
apim api grant-access    Grant access to selected APIs to the given organization.
```

You would need to give the Org-Id or Organization-Name

**Comment by tcheronneau** [comment]
Hey thanks for your answer. 
I'm no expert to axway but for me the notion of grant-access start at API side and give access to an org. 
So imho it should be part of api section.

**Comment by tcheronneau** [comment]
@cwiechmann I've built the lib in order to test this part. 
I don't know if it's yet fully functional but with my latest build (pulled this morning) the --api-path option does not seems to be taken into account. 
Example : 
```
apim api grant-access -orgName TEST -a /bg/v2 -force
/usr/bin/java

----------------------------------------------------------------------------------------
API-Manager CLI: 1.3.7-SNAPSHOT

To report issues or get help, please visit:
https://github.com/Axway-API-Management-Plus/apim-cli
----------------------------------------------------------------------------------------
Module: API - E X P O R T / U T I L S (1.3.7-SNAPSHOT)
----------------------------------------------------------------------------------------
126    INFO     APIManagerAdapter| Successfully connected to API-Manager (7.7.20200930) on: https://localhost:20176
388    INFO          APIExportApp| 2 API(s) and 1 Organization(s) selected.
Selected organizations: ['TEST'] get access to 2 selected APIs.
Force flag given to grant access for selected organizations to: 2 API(s)
Okay, going to grant access to: 2 API(s) for 1 organizations.
430    INFO      APIResultHandler| API: TEST1 (1.13220190215.037.023) exposed on path: /bg/v1 granted access to orgs: ['TEST']
449    INFO      APIResultHandler| API: TEST2 (1.4.0) exposed on path: /api/v1.4 granted access to orgs: ['TEST']
```

**Comment by cwiechmann** [comment]
Yes, that looks wrong.

**Comment by cwiechmann** [comment]
@tcheronneau, the issue is fixed (#164) and will be released with version 1.3.7. Feel free to re-build it on your side.

**Comment by tcheronneau** [comment]
Seems better now thanks !
